src/SqlPersistence/Config/ConnectionManager/SingleTenantConnectionManager.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.Common;
+using System.Threading.Tasks;
+
+class SingleTenantConnectionManager : ConnectionManager
```

Comment:
Is the SingleTenantConnectionManager the one used regularly in non multi-tenant environments? If yes class name is misleading (at least at a first look)

Reply:
Yes. At one point I had `MultiTenantConnectionManager : AbstractConnectionManager` and `ConnectionManager : AbstractConnectionManager` but then having `AbstractConnectionManager`s littered all throughout the consuming code just looked yucky.